lib/constants.py

```diff
@@ -93,6 +93,34 @@ class BitcoinTestnet:
     }
 
 
+class BitcoinRegtest:
```

Comment:
Actually, I'm thinking it might be better to inherit from `BitcoinTestnet` so some of the fields don't need to be duplicated :P

Reply:
No problem :)